api/wordReaderService.ts

```diff
@@ -0,0 +1,34 @@
+import got from "got";
+
+const jsdom = require("jsdom");
+const { JSDOM } = jsdom;
+
+const readGeneratedName = async (): Promise<string> => {
+	const vgmUrl= 'https://www.classnamer.org/';
+
+	try {
+		const response = await got(vgmUrl)
+		const dom = new JSDOM(response.body);
+		const nameValue = dom?.window?.document?.getElementById('classname').innerHTML;
+
+		console.log('name value: ', nameValue);
+		return nameValue;
+	} catch(err) {
+		console.error('error while reading a generated name', err);
```

Comment:
You should handle different kinds of errors differently.
The page in unreachable/the classname element doesn't exist.. and maybe log a different error for each